cmd/notation/blob/inspect.go

```diff
@@ -20,14 +20,15 @@ import (
 
 	"github.com/notaryproject/notation-core-go/signature"
 	"github.com/notaryproject/notation/cmd/notation/internal/display"
-	"github.com/notaryproject/notation/cmd/notation/internal/option"
+	"github.com/notaryproject/notation/cmd/notation/internal/display/output"
+	"github.com/notaryproject/notation/cmd/notation/internal/flag"
 	"github.com/spf13/cobra"
 )
 
 type inspectOpts struct {
-	option.Common
-	option.Format
-	sigPath string
+	outputFormat  flag.OutputFormatFlagOpts
+	outputPrinter *output.Printer
```

Comment:
is it simpler to just call it a printer?
```suggestion
	printer *output.Printer
```

Reply:
Nope, it needs to be `outputPrinter` as it indicates a printer for output only.

Reply:
In case a command also prints out other staff in addition to output, maybe `displayPrinter` is more accurate.